engine-types/src/parameters/connector.rs

```diff
@@ -165,6 +165,13 @@ pub struct StorageUnregisterCallArgs {
 #[derive(Debug, Clone, BorshSerialize, BorshDeserialize, PartialEq, Eq)]
 pub struct SetEthConnectorContractAccountArgs {
     pub account: AccountId,
+    pub withdraw_serialize_type: WithdrawSerializeType,
```

Comment:
Should maybe be optional with default value `WithdrawSerializeType::Borsh` for backwards compatibility

Reply:
I thought that we don't have any production contract with such functionality and therefore I skipped it. But why not?

Reply:
By the way. I'm not sure that borsh allows us to use default value if there are no bytes for field like serde does. I mean that we can't get: 
```rust
SetEthConnectorContractAccountArgs {
    account: "eth-connector.near".parse().unwrap(),
    withdraw_serialize_type: WithdrawSerializeType::Borsh,
}
```
from
```
 "eth-connector.near".parse::<AccountId>().unwrap().try_to_vec().unrwap()
```

If only to write custom deserialization for SetEthConnectorContractAccountArgs. But does it make sense? As I said we don't use `set_eth_connector_contract_account` api in production.

Reply:
Yeah, I agree it doesn't make sense to put in so much work when there are no production silos that use it yet.